src/isomorphic/hooks/ReactComponentTreeHook.js

```diff
@@ -110,7 +120,7 @@ function purgeDeep(id) {
 }
 
 function describeComponentFrame(name, source, ownerName) {
-  return '\n    in ' + name + (
+  return '\n    in ' + (name || 'Unknown') + (
```

Comment:
While trying to fix flow, I ended up moving that inline check here. I can revert it